<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>База данных аудиторий и технического оборудования</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="rooms_list">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>200</width>
      <height>700</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="equipment_info">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>540</width>
      <height>350</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="equipment_image">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>380</y>
      <width>540</width>
      <height>340</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="previous_button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>730</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QPushButton" name="next_button">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>730</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Вперед</string>
    </property>
   </widget>
   <widget class="QLabel" name="computer_count_label">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>730</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Компьютеры: 0/0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>